js/components/screenGroupsList.js

```diff
@@ -30,9 +41,16 @@ window.ScreenGroupList = React.createClass({
     );
   },
   renderListOfGroups: function () {
+    var jsonGroups = this.props.getGroups();
+    var arrGroups = jsonGroups.map(function (o) {
+      // here setRoute is not working..not in scope? do we need state here?
+      //  neede to pass change Route..and pass groupId..to show posts of groupId
+      // return <Group groupObj={o} setRoute={this.props.setRoute} />;
+      return <Group groupObj={o} />;
```

Comment:
This issue has to do with how `this` is evaluated in this function. What you can do is declare 
```js
var setRoute = this.props.setRoute
```
outside the function and and then use it.